babel.config.js

```diff
@@ -0,0 +1 @@
+module.exports = {};
```

Comment:
what does this do?

Reply:
an update of one of these dependencies required the existence of a babel.config.js. sorry i don't remember which. consuming repos like frontend specify one but this project linting itself needed one too with these changes.

Reply:
ok cool!